proprietary-files.txt

```diff
@@ -17,6 +17,7 @@ vendor/lib/libadiertac.so
 vendor/lib/libaudcal.so
 vendor/lib/libhwdaphal.so
 vendor/lib/libqct_resampler.so
+vendor/lib/libvoice-svc.so
```

Comment:
This list should be updated according to https://github.com/boulzordev/android_vendor_lenovo_A6020/pull/4/